api/src/main/java/io/opentelemetry/metrics/DoubleValueRecorder.java

```diff
@@ -16,39 +16,54 @@
 
 package io.opentelemetry.metrics;
 
-import io.opentelemetry.metrics.DoubleMeasure.BoundDoubleMeasure;
+import io.opentelemetry.metrics.DoubleValueRecorder.BoundDoubleValueRecorder;
 import java.util.Map;
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * Measure to report instantaneous measurement of a double value.
+ * ValueRecorder is a non-additive synchronous instrument useful for recording any non-additive
+ * number, positive or negative. Values captured by a Record(value) are treated as individual events
+ * belonging to a distribution that is being summarized.
+ *
+ * <p>ValueRecorder should be chosen either when capturing measurements that do not contribute
+ * meaningfully to a sum, or when capturing numbers that are additive in nature, but where the
+ * distribution of individual increments is considered interesting.
+ *
+ * <p>One of the most common uses for ValueRecorder is to capture latency measurements. Latency
+ * measurements are not additive in the sense that there is little need to know the latency-sum of
```

Comment:
ah, I see "non-additive" is clarified here. I'm still not sure usage of this terminology adds a lot to the user's decision to use a ValueRecorder.

Reply:
I left this here, let me know if you think we should remove.

Reply:
I think is fine to leave, yes.